### DISCUSSION TITLE
Upgrade to 2.5.301 -> Error: getaddrinfo EAI-AGAIN wiki-update-companion

### DISCUSSION THREAD

**Discussion by Zymonick** [requarks/wiki#7003] [Error / Bug Report]
Hello together

I run a relatively fresh wiki.js installation on version 2.5.300. In the administration area, I got prompted that there is an upgrade available. However, when I click on upgrade I get the rather cryptic error message: 
"Error: getaddrinfo EAI_AGAIN wiki-update-companion"

![image](https://github.com/requarks/wiki/assets/16645985/d24baa9f-c3c7-4adb-a1b0-309d09b19f1e)

Is this a bug or is there something on my side in the installation that I need to do to get this to work?

**Comment by mike858585** [comment]
Maybe some dns problem with your docker-composer related to https://github.com/requarks/wiki/discussions/3401 . I use the installation via helm-chart so the only problem I had with the update was with the set low resources.
